### ISSUE TITLE
Waiting for data results feedback

### ISSUE THREAD

**Issue by tassoman** [GuillaumeAmat/leaflet-overpass-layer#16]
Ciao,
 is there any chance to get someting scrobbling or a notification while data are being fetched and onSuccess isn't yet triggered? Sometimes it happens the query is slow, the effect is missing markers in the map.
Thank you for your time :smile:

**Comment by GuillaumeAmat** [comment]
Hi,
I'm not sure to understand what you want... You would like a sort of *progress* callback? It's not something I can implement as the Overpass response comes in a single chunk.
You can use the *beforeRequest* callback though.

**Comment by tassoman** [comment]
Do you suggest to show [a tooltip](http://leafletjs.com/reference-1.0.3.html#tooltip) on **beforeRequest** then hiding it _onSuccess_ or updating _onError_ ?

**Comment by GuillaumeAmat** [comment]
Oh I just understand, you would like a piece of interface displaying a loader or something?
It could be great indeed, but I'm running out of time to do that currently... Can you consider to make a pull request?

**Comment by tassoman** [comment]
Yeah! Sorry for being unclear but I was in a full immmersion, two days coding and my mind went a bit crazy... :unamused: 
I've solved adding a `<div id="wait">Loading...</div>`, its CSS is a whole transparent grey page with a font-awesome scrobbler.

I managed to set its `z-index` _beforeRequest_ `$('#wait').css('z-index',9999);` and _afterRequest_ `$('#wait').css('z-index',-1);` by `$('#wait').css('z-index',9999);`.

For now it's barely working, giving feedback on requesting times, with no evidence of faulty or successful results.
